apps/opencs/view/render/terraintexturemode.cpp

```diff
@@ -418,18 +568,21 @@ void CSVRender::TerrainTextureMode::createTexture(std::string textureFileName)
 
     int counter=0;
     bool freeIndexFound = false;
-    do {
+    do
+    {
         const size_t maxCounter = std::numeric_limits<uint16_t>::max() - 1;
         try
         {
             newId = CSMWorld::LandTexture::createUniqueRecordId(0, counter);
             if (ltexTable.getRecord(newId).isDeleted() == 0) counter = (counter + 1) % maxCounter;
-        } catch (const std::exception& e)
+        }
+        catch (const std::exception& e)
         {
             newId = CSMWorld::LandTexture::createUniqueRecordId(0, counter);
             freeIndexFound = true;
         }
-    } while (freeIndexFound == false);
+    }
+    while (freeIndexFound == false);
```

Comment:
You probably shouldn't touch the formatting of this function, it's not that outrageous.

Reply:
Yeh. Not very significant maybe, but changed it back.